src/etcpal/pack.c

```diff
@@ -22,12 +22,54 @@
 
 /*************************** Function definitions ****************************/
 
+/*!
+ * \brief Unpack an int16_t from a known big-endian buffer.
+ * \param buf Pointer to the buffer from which to unpack a value; must be at least 2 bytes in size.
+ * \return Unpacked value.
+ */
+int16_t etcpal_unpack_i16b(const uint8_t* buf)
+{
+  // TODO
```

Comment:
We should really do the signed versions of packing/unpacking sooner rather than later.